### DISCUSSION TITLE
Broken Quiz: 3.2 The Logistic Regression Computation Graph

### DISCUSSION THREAD

**Discussion by Diego74965** [Lightning-AI/dl-fundamentals#14] [General discussions]
Hi, in Quiz 3.2, you are not able to select the correct answer. Plus other formatting errors
![image](https://user-images.githubusercontent.com/33135679/210117545-8a09fd63-49c5-4aac-a6c3-c346f888ab7f.png)

**Comment by rasbt** [comment]
Thanks for the note @Diego74965, I will ping our webdev about that next week!

**Comment by Diego74965** [comment]
This problem is also present in Quiz 3.3 part 1

**Comment by rasbt** [comment]
The quiz formatting should be fixed now 😊
